tests/Tests/600__ai_apps/640__rhoam/640__rhoam_installation.robot

```diff
@@ -0,0 +1,53 @@
+*** Settings ***
+Resource        ../../../Resources/Page/LoginPage.robot
+Resource        ../../../Resources/Common.robot
+Resource        ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+Resource        ../../../Resources/Page/ODH/AiApps/Rhoam.resource
+Resource        ../../../Resources/ODS.robot
+Library         ../../../../libs/Helpers.py
+Library         SeleniumLibrary
+Library         OpenShiftLibrary
+Suite Setup     RHOAM Install Suite Setup
+Suite Teardown  RHOAM Suite Teardown
+
+
+*** Test Cases ***
+Verify RHOAM Can Be Installed
+    [Documentation]    Verifies RHOAM Addon can be successfully installed
+    [Tags]  Tier3
+    ...     ODS-273
+    ...     Execution-Time-Over-30min
+    Pass Execution    Passing tests, as suite setup ensures that RHOAM installs correctly
+
+Verify RHODS Can Be Uninstalled When RHOAM Is Installed
+    [Documentation]    Verifies RHODS can be successfully uninstalled when
+    ...                RHOAM addon is installed on the same cluster
+    [Tags]  Tier3
+    ...     ODS-1136
+    ...     DestructiveTest
+    ...     Execution-Time-Over-2h
+    Verify RHOAM Is Enabled In RHODS Dashboard
+    Uninstall RHODS From OSD Cluster
```

Comment:
If _Uninstall RHODS From OSD Cluster_  doesn't support the add-on flow (I'm not sure about that) please consider adding the _Deployment-Cli_ tag to the test

Reply:
for the moment I've disabled the addon flow, but the OLM uninstall works fine in both cases. The only bad looking aspect is that if you run it on a Addonflow installation, the OCM UI will still show "installed" as status